Makefile

```diff
@@ -97,7 +97,7 @@ help-test :
 	@echo "    (see also setup.cfg's pytest configuration)"
 
 test : $(STATEDIR)/docker-build
-	docker-compose run testing bin/test $(TEST_EXTRA_ARGS) $(TEST)
+	docker-compose run testing bin/test $(filter-out $@, $(MAKECMDGOALS)) $(TEST_EXTRA_ARGS) $(TEST)
```

Comment:
FYI these two changes allow this syntax:
make test <path/to/test>
and even:
make test -- <path/to/test> --pdb
to parallel
 python setup.py <path/to/test> --pdb